documentation/sdkautomation/sdk_customization.md

```diff
@@ -779,23 +290,35 @@ Output of install instruction script.
 
 #### InstallInstructionScriptOutput Schema
 
+See [./InstallInstructionScriptOutput.json](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/sdkautomation/InstallInstructionScriptOutput.json)
+
+### TriggerType
+
+#### TriggerType Schema
+
+```jsonc
```

Comment:
Why did you keep this schema inline where you pulled out all the others? Is there a reason you pulled out the other schemas? Are you planning to consume them in some way? I will say that probably makes reading the spec a little more difficult because you have to jump around a lot to see the different things.

Reply:
I will add perhaps just having examples, like you do for most things, instead of the schema might be enough for the spec.

Reply:
I organize the schema json in my project and copy them out so that I could easily sync it between code and doc, and user could reference the json schema directly to have editor support.

Reply:
It's not very easy to merge everything into one json schema file because my code also consume these schemas

Reply:
I wasn't suggested that you create one schema file I think having one schema file for each part is fine so that you can use it for the tools. Just provide a link to the schema and an example in the md documentation.